src/intervals/interval_operations/pointwise_boolean.jl

```diff
@@ -1,39 +1,27 @@
 # This file is part of the IntervalArithmetic.jl package; MIT licensed
 
-#=  This file contains functions not specified in the IEEE Std 1788-2015,
-    extending boolean operations defined for reals to intervals.
-
-    This is different from what is described in sections 9.5 of the
-    IEEE Std 1788-2015 (Boolean functions of intervals), since we deal here
-    with what is needed for drop-in replacement of floats by intervals,
-    and not what is reasonnable to do with intervals in a separate execution
-    environment.
-
-    Essentially, julia is better at composability and we are more ambitious
-    that what the standard ever expected, so we have some extra problems
-    to solve.
+#=  This file contains boolean operations defined for intervals.
```

Comment:
I think we should adapt part of the explanation of @Kolaru that was removed here. First, the `PointwisePolicy` does allow to consider other behavior for the boolean functions on intervals, which does not follow the specifications of the IEEE Std 1788-2015. (This may be interpreted as another flavor, e.g. cset + PoinwisePolicy.) And secondly, because Benoit is right, Julia is better at composability and we are ambitious :-)